docs/getting-started/integrate-auth/10_expressjs.mdx

```diff
@@ -65,7 +65,7 @@ Now that the Ory CLI is installed, create a new npm script to run the proxy serv
   "private": true,
   "scripts": {
     "dev": "nodemon index.js",
-    "proxy": "ory proxy --project $ORY_PROJECT_ID --port 4000 http://localhost:3000/"
+    "tunnel": "npx @ory/cli tunnel --dev http://localhost:3000",
```

Comment:
Not in this PR, but the npm package `@ory/cli` is not great because it uses binwrap which has some unpatched vulnerabilities. Ideally, we would install the ory cli using the same install approach as for the other languages. Maybe you can create an issue to track this